### PR TITLE
Add Speedpool to pool.json

### PR DESCRIPTION
**Pull request by popof1664**
Add Speedpool to Zelcash Explorer
Thanks

### CODE REVIEW

**Comment by TheTrunk** [comment]
Hello @popof1664.
The searchString shall be your pool address, please correct the PR. 
Thank you.

**Comment by popof1664** [comment]
hello @TheTrunk 
Ok thanks, i will change SearchString to equihash.speedpool.top

File: pools.json [hunk 1]
```diff
@@ -188,5 +188,12 @@
         "searchStrings": [
             "t1cVYU7yj2doV33B1Acn586FXn2GFSuwi3f"
         ]
+    },
+    {
+    	"poolName": "Speedpool",
+    	"url": "https://equihash.speedpool.top/getting_started",
+    	"searchStrings": [
+      	    "equihash.speedpool.top"
```
Comment (TheTrunk) [inline pools.json]: The search string has to be your zelcash pool address. Address to which your pool is mining to.

**Review by TheTrunk — CHANGES_REQUESTED** [review]

File: pools.json (hunk 1 above)
Comment (popof1664) [reply pools.json]: yes it is, to mine zelcash the url = stratum+tcp://equihash.speedpool.top: 8043
The searchstring parameter is = url = equihash.speedpool.top

Comment (TheTrunk) [reply pools.json]: Search string parameter has to be your zelcash transparent address - starts with t1, look at many examples above your addition. Insight explorer is looking for an address to which the coins were mined to.

**Pull request merged**